nav2_behavior_tree/include/nav2_behavior_tree/plugins/control/round_robin_node.hpp

```diff
@@ -30,7 +30,8 @@ namespace nav2_behavior_tree
  *  RoundRobin           |    Tick Next Child    | Return Running
  *
  * If the current child return failure, the next child is ticked and if the last child returns
- * failure, the first child is ticked and the cycle continues until a child returns success
+ * failure, the first child is ticked and the cycle continues until a child returns success.
```

Comment:
Please add this change to the galactic migration guide so that people know we changed behavior